uaclient/contract.py

```diff
@@ -189,23 +189,83 @@ def _get_platform_data(self, machine_id):
         return {"machineId": machine_id, "architecture": arch, "os": platform}
 
 
-def process_entitlement_delta(orig_access, new_access, allow_enable=False):
+def process_entitlements_delta(
+    past_entitlements: "Dict[str, Any]",
+    new_entitlements: "Dict[str, Any]",
+    allow_enable: bool,
+    series_overrides: bool = True,
+) -> None:
+    """Iterate over all entitlements in new_entitlement and apply any delta
```

Comment:
thanks for the docs. Saves a lot of brain power in reviews and future maintenance